packages/core/src/EventManager.ts

```diff
@@ -28,6 +35,7 @@ export class EventManager {
      */
     _onNormalizedEvent(customEvent: CustomEvent): void {
         const payload = customEvent.detail;
```

Comment:
Damn it, this is (obviously) not updated according to the `EventNormalizer` branch ! (I just realized that's what was actually blocking this PR.)